lib/JobLogs/tests/jobLogs-test.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { expect } from 'chai';
+import {
+  describe,
+  beforeEach,
+  it,
+} from '@bigtest/mocha';
+
+import MultiColumnListInteractor from '@folio/stripes-components/lib/MultiColumnList/tests/interactor';
+
+import translation from '../../../translations/stripes-data-transfer-components/en';
+import { mountWithContext } from '../../../test/bigtest/helpers';
+import PreloaderInteractor from '../../Preloader/tests/interactor';
+import getItemFormatter from '../getItemFormatter';
+import { jobsLogs } from '../../tests/jobsLogs';
+import JobLogs from '../JobLogs';
+
+const getJobLogsComponent = (formatter, hasLoaded, contentData) => (
+  <Router>
+    <div style={{
+      height: '1000px',
+      width: '1000px',
+    }}
+    >
+      <JobLogs
+        formatter={formatter}
+        hasLoaded={hasLoaded}
+        contentData={contentData}
+      />
+    </div>
+  </Router>
+);
+
+describe('Logs list', () => {
+  const logsList = new MultiColumnListInteractor('#job-logs-list');
+  const preloaderInteractor = new PreloaderInteractor();
+  const getCellContent = (row, cell) => logsList.rows(row).cells(cell).content;
+
+  describe('Rendering not loaded job logs with default props', () => {
+    beforeEach(async () => {
+      await mountWithContext(getJobLogsComponent(
+        getItemFormatter('/link/'),
+        false,
+        []
+      ));
+    });
+
+    it('should not display logs lists', () => {
+      expect(logsList.isPresent).to.be.false;
+    });
+
+    it('should display preloader', () => {
+      expect(preloaderInteractor.isPresent).to.be.true;
+    });
+  });
+
+  describe('Rendering loaded job logs with default props and empty logs list', () => {
```

Comment:
```suggestion
  describe('rendering loaded job logs with default props and empty logs list', () => {
```
for consistency starting with lowercase, what do you think?